components/VcItemContent.tsx

```diff
@@ -13,33 +13,43 @@ import testIDProps from '../shared/commonUtil';
 const getDetails = (arg1, arg2, verifiableCredential) => {
   if (arg1 === 'Status') {
     return (
-      <Column>
+      <Column
+        style={{
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          marginTop: 7,
+        }}>
         <Text
           testID="status"
-          weight="bold"
-          size="smaller"
```

Comment:
Here we are passing **`size`** as a prop which is already implemented in the Text.tsx component. From there it checks from the themes.

Reply:
Done